Makefile

```diff
@@ -429,6 +437,14 @@ bundle-build: ## Build the bundle image.
 bundle-push: ## Push the bundle image.
 	$(OCI_BIN) push ${BUNDLE_IMAGE};
 
+.PHONY: file-based-catalog-build
```

Comment:
just curious why did  u added those two new targets since konflux pipeline will use the containerfile directly ?

Reply:
We currently have target to build a regular catalog, but if I understand correctly at some point this catalog will be replaced by file based catalog.

I wanted to introduce the new targets without replacing the old ones for now.